### DISCUSSION TITLE
does anyone know a bypass for BARK?

### DISCUSSION THREAD

**Discussion by nondious** [3kh0/ext-remover#559] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
my school has a really shitty extension called 'Bark for Schools Chrome Filter' and using the task manager to shut it down doesn't work, and it automatically starts back up again. just wondering if anyone knows how to bypass this? it blocks so many sites and it makes everyone mad 🤣

**Comment by obkduishinijf** [comment, marked as answer]
point blank, LTBEEF on version 101 and below, and that's all i know.
